doc/faq.md

```diff
@@ -47,6 +47,18 @@ MPI support may be added in the future if sponsored by someone. If you would lik
 
 ***
 
+### Python scripts that import openquake
+
+On **Ubuntu** and **RHEL** if a third party python script (or a Jupyter notebook) needs to import openquake as a library (as an example: `from openquake.commonlib import readinput`) you must configure the `PYTHONPATH` first:
+
+```bash
+export PYTHONPATH="/opt/openquake/lib/python2.7/site-packages:$PYTHONPATH"
+```
```

Comment:
This does not apply for our cluster, but only when using official packages for Ubuntu and RHEL